packages/suite/src/views/wallet/coinmarket/common/CoinmarketPaymentPlainType.tsx

```diff
@@ -0,0 +1,50 @@
+import { ReactNode } from 'react';
+import styled from 'styled-components';
+import { variables } from '@trezor/components';
+import { BuyCryptoPaymentMethod, SavingsPaymentMethod, SellCryptoPaymentMethod } from 'invity-api';
+import { Translation } from 'src/components/suite';
+
+const Text = styled.div`
+    display: flex;
+    align-items: center;
+    font-size: ${variables.FONT_SIZE.NORMAL};
+    color: ${({ theme }) => theme.TYPE_DARK_GREY};
+    font-weight: ${variables.FONT_WEIGHT.MEDIUM};
+`;
```

Comment:
We recently had a big redesign and for now on we don't use: 
1. Font weight and size from variables, instead pick some type from `@trezor/theme` typography. This one would probably be`${typography.body}`.
2. Upper case colors, instead use camelCase colors since there are updated with current brand. This one would probably be `theme.textSubdued`